monai/data/utils.py

```diff
@@ -399,3 +401,40 @@ def create_file_basename(postfix, input_file_name, folder_path, data_root_dir=""
 
     # add the sub-folder plus the postfix name to become the file basename in the output path
     return os.path.join(subfolder_path, filename + "_" + postfix)
+
+
+def compute_importance_map(patch_size, mode='constant', sigma_scale=0.125, device=None):
+    """Get importance map for different weight modes.
+
+    Args:
+        patch_size (tuple): Size of the required importance map. This should be either H, W [,D].
+        mode (str): Importance map type. Options are 'constant' (Each weight has value 1.0)
+            or 'gaussian' (Importance becomes lower away from center).
+        sigma_scale (float): Sigma_scale to calculate sigma for each dimension 
+            (sigma = sigma_scale * dim_size). Used for gaussian mode only.
+
+    Returns:
+        numpy array of size patch_size.
+    """
+    importance_map = None
+    if mode == 'constant':
+        importance_map = torch.ones(patch_size, device=device).float()
+    elif mode == 'gaussian':
+        center_coords = [i // 2 for i in patch_size]
+        sigmas = [i * sigma_scale for i in patch_size]
+
+        importance_map = torch.zeros(patch_size, device=device)
+        importance_map[tuple(center_coords)] = 1
+        pt_gaussian = GaussianFilter(len(patch_size), sigmas)
```

Comment:
missing `to(device)` for the `GaussianFilter`. It should be:
```py
gaussian_filter = GaussianFilter(len(patch_size), sigmas).to(device)
```

Reply:
Added.